GPU/Common/GPUStateUtils.cpp

```diff
@@ -1058,8 +1058,10 @@ static void ConvertBlendState(GenericBlendState &blendState, bool forceReplaceBl
 
 	ReplaceBlendType replaceBlend = ReplaceBlendWithShader(gstate_c.framebufFormat);
 	if (forceReplaceBlend) {
-		replaceBlend = REPLACE_BLEND_READ_FRAMEBUFFER;
+		// Enforce blend replacement if enabled. If not, shouldn't do anything of course.
+		replaceBlend = blendState.blendEnabled ? REPLACE_BLEND_READ_FRAMEBUFFER : REPLACE_BLEND_NO;
```

Comment:
Isn't this function responsible for populating this flag, lower down?  Is it even valid at this point?  It would just be the previous blendEnabled, which is irrelevant since state changed?

-[Unknown]